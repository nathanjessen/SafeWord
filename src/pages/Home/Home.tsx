import FeatureSection from '../../components/FeatureSection';
import Gallery from '../../components/Gallery';
import Hero from '../../components/Hero';
import Roadmap from '../../components/Roadmap';
import SafeWordGenerator from '../../components/SafeWordGenerator';
import Team from '../../components/Team';

export const Home = () => {
  return (
    <main>
      <Hero />
      <SafeWordGenerator />
      <Gallery />
      <Roadmap />
      <FeatureSection />
      <Team />
    </main>
  );
};
