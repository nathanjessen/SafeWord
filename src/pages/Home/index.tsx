import React from "react";

export default function Home() {
  return (
    <main className='min-w-0 flex-1 flex items-center justify-center overflow-y-scroll'>
      <section
        aria-labelledby='primary-heading'
        className='px-4 sm:px-8 max-w-8xl'>
        <h1 id='primary-heading' className='sr-only'>
          Home
        </h1>
        <h1 className='text-white text-8xl text-center'>SafeWord</h1>
        <h1 className='text-gray-200 text-5xl text-center'>Coming Soon</h1>
      </section>
    </main>
  );
}
