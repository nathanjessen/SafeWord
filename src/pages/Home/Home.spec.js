import ReactDOM from "react-dom";
import { cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import Home from "./index";

afterEach(cleanup);

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Home />, div);
});

test("matches snapshot", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
