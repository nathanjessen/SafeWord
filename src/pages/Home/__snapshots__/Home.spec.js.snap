// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="p-4 lg:p-6"
>
  <p>
    Nothing to display
  </p>
</div>
`;
